pkg/option/config.go

```diff
@@ -2191,6 +2191,11 @@ func (c *DaemonConfig) NodeEncryptionEnabled() bool {
 	return c.EncryptNode
 }
 
+// EncryptionEnabled returns true if node encryption is enabled
```

Comment:
Is this comment correct? If I understand this right then we essentially decoupled `Encryption` from `NodeEncryption` right?

Reply:
no its not correct fixed by dropping node encryption and replacing with just "encryption"